CHANGES.md

```diff
@@ -1,3 +1,62 @@
+# Version 0.5
+
+## New Features
+
+* Added experimental support for basic, non-compositional verification
+  of x86 machine code for use in conjunction with LLVM verification.
+
+        crucible_llvm_verify_x86 :
+          LLVMModule -> String -> String ->
+          [(String, Int)] -> Bool -> CrucibleSetup () ->
+          TopLevel CrucibleMethodSpec
+
+
+  The first argument specifies the LLVM module containing the _caller_.
```

Comment:
this text and the one in `Interpreter.hs` should match

Reply:
Yep. I'll update `Interpreter.hs`, since it's less complete.